dandiapi/sitemaps.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from django.conf import settings
+from django.contrib.sitemaps import Sitemap
+
+from dandiapi.api.models.dandiset import Dandiset
+
+
+class DandisetSitemap(Sitemap):
+    changefreq = 'weekly'
+    priority = 0.5
+
+    def items(self):
+        return Dandiset.objects.filter(embargo_status=Dandiset.EmbargoStatus.OPEN)
+
+    def location(self, obj: Dandiset) -> str:
+        return f'/dandiset/{obj.identifier}/'
+
+    def get_domain(self, site):
+        return settings.DANDI_WEB_APP_DOMAIN
```

Comment:
Since this is only used in one place, maybe we can avoid adding a new setting:
```suggestion
        return urlparse(settings.DANDI_WEB_APP_URL).netloc
```

Reply:
Good idea; see 25352f5b2dc6f8d7bf36b14a4fd6d65e7a1c3290.